test/e2e/reboots/operator_test.go

```diff
@@ -19,7 +19,7 @@ var (
 func TestNodeTuningOperator(t *testing.T) {
 	gomega.RegisterFailHandler(ginkgo.Fail)
 
-	ginkgo.By("getting cluster ControlPlaneTopology")
+	util.Logf("getting cluster ControlPlaneTopology")
```

Comment:
why moving to `util.Logf`?

Reply:
this is a change from @jmencak but I think there a By cause a crash, change for util.Log you get the log and the test works. @jmencak could you correct my comment?

Reply:
I'm fine moving to `util.Logf`, I'm just curious about the rationale

Reply:
As I remember after change to ginkgo v2 that crashes because By cannot be outside a ginkgo node